### DISCUSSION TITLE
Equation of ResGatedGraphConv Operator when Edge Features are Included

### DISCUSSION THREAD

**Discussion by liujiale-study** [pyg-team/pytorch_geometric#8826] [General]
Hello, I am asking with regards to the ResGatedGraphConv Operator 
(https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.nn.conv.ResGatedGraphConv.html)

The equation in the documentation didn't mention edge features although we can pass them into the operator.
May I know the what the equation is like when edge features are added into the computation?

Thanks in advance!

**Comment by rusty1s** [comment]
Sorry about that. For the edge feature case, we concatenate the edge features to the query/key/value features, e.g., it becomes
```
n_ij * W_2(x_j || edge_feat)
```
where 
```
n_ij = sigma(W_3(x_i || edge_feat) + W_4(x_j || edge_feat))
```

**Reply by liujiale-study** [reply]
Thanks alot!
